recipes-webadmin/cockpit-config/cockpit-config.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "Cockpit configuration for Seapath"
```

Comment:
Why create a cockpit-config recipe ?
It is simpler to put it directly in meta-seapath cockpit bbappend